src/index.js

```diff
@@ -225,6 +250,9 @@ function setAnalyticsIntegration(clientInstance) {
   return clientInstance;
 }
 
+// we check for document's existence to support server side rendering
+createFromScriptTag(create, document && document.querySelector('script[data-braintree-dropin-authorization]'));
```

Comment:
This will give a `ReferenceError` in server-side rendering. I believe the expression should be:

```js
typeof document !== 'undefined' && document.querySelector('script[data-braintree-dropin-authorization]')
```

I could see throwing a `DropinError` if it finds multiple scripts.